src/ashok/A15_array_assignment.java

```diff
@@ -0,0 +1,100 @@
+package ashok;
+
+public class A15_array_assignment {
+	
+	int maxArrayNumber(int [] num) {
+		int maxnumber = 0;
+		for(int index = 0 ; index<num.length; index++) {
```

Comment:
If you are taking the 0th index as max number then you can start the loop from index 1 as well, same for other methods